src/constants.ts

```diff
@@ -37,3 +37,5 @@ export const envFileName: string = '.env';
 export const detectorTimestampFormat: string = 'YYYY-MM-DDTHH:mm';
 
 export const webProvider: string = 'Microsoft.Web';
+
+export const sshURL: string = 'root@127.0.0.1';
```

Comment:
Since this is only used in startSsh.ts you could declare it in startSsh.ts instead of exporting it here.